lib/ticks.js

```diff
@@ -63,7 +63,15 @@ function ticksEqual(ticksA, ticksB) {
     for(var j=0; j<ticksA[i].length; ++j) {
       var a = ticksA[i][j]
       var b = ticksB[i][j]
-      if(a.x !== b.x || a.text !== b.text) {
+      if(
+        a.x !== b.x ||
+        a.text !== b.text ||
+        a.font !== b.font ||
+        a.fontColor !== b.fontColor ||
+        a.fontSize !== b.fontSize ||
+        a.dx !== b.dx ||
+        a.dy !== b.dy
```

Comment:
A bit repetitive but it's in a potentially perf critical code part - e.g. now runs on each hover move - and there may be internal properties too so I ended up not `JSON.stringify`ing it or adding a loop on `Object.keys`